composer.json

```diff
@@ -38,8 +36,12 @@
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.15",
         "localheinz/composer-normalize": "~0.8.0",
+        "php-http/httplug": "^1.0 || ^2.0",
+        "php-http/message-factory": "^1.0.2",
         "phpbench/phpbench": "@dev",
         "phpunit/phpunit": "^6.0",
+        "psr/http-client": "^1.0",
```

Comment:
Since it will be next usual way of generating Jane clients, I will prefer to have it in require section.